08-rails/installing-foundation.md

```diff
@@ -0,0 +1,120 @@
+#  Installing Foundation By Default
+
+## Learning Goals
+- Practice modifying our `.rails-template.rb` file to add an additional gem and content.
+- Learn how to install and use the Foundation Framework via the gem.
```

Comment:
Should we reverse the order of these learning goals so they match the order in which the material is presented?